src/main/java/org/primeframework/mvc/cors/CORSConfiguration.java

```diff
@@ -40,6 +41,12 @@ public class CORSConfiguration implements Buildable<CORSConfiguration> {
 
   public Pattern excludedPathPattern;
 
+  public Pattern includedPathPattern;
+
+  public Predicate<String> excludeUriChecker;
+
+  public Predicate<String> includeUriChecker;
```

Comment:
Naming: `excludeURIPredicate;` and `includeURIPredicate`

Reply:
Done